pkg/kubelet/container/runtime.go

```diff
@@ -346,6 +346,18 @@ func (podStatus *PodStatus) FindContainerStatusByName(containerName string) *Sta
 	return nil
 }
 
+// GetAllContainerStatusesByName return all running container status in the pod status with the given name.
+// the containers have already been sorted by CreatedAt
+func (podStatus *PodStatus) GetAllContainerStatusesByName(containerName string) []*Status {
```

Comment:
This is almost a duplicate of `FindContainerStatusByName`:

https://github.com/kubernetes/kubernetes/blob/032a6fd49429f5d09814c43a243ea74f6c64cbe9/pkg/kubelet/container/runtime.go#L338-L340

We should probably be consistent in the naming.